packages/babel-parser/src/plugins/flow.js

```diff
@@ -2823,4 +2861,416 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         this.raise(this.state.pos, "Unterminated comment");
       }
     }
+
+    // Flow enum parsing
+
+    flowEnumErrorBooleanMemberNotInitialized(
+      pos: number,
+      { enumName, memberName }: { enumName: string, memberName: string },
+    ): void {
+      this.raise(
+        pos,
+        `Boolean enum members need to be initialized. Use either \`${memberName} = true,\` ` +
+          `or \`${memberName}\` = false,\` in enum \`${enumName}\`.`,
+      );
+    }
+
+    flowEnumErrorInvalidMemberName(
+      pos: number,
+      { enumName, memberName }: { enumName: string, memberName: string },
+    ): void {
+      const suggestion = memberName[0].toUpperCase() + memberName.slice(1);
+      this.raise(
+        pos,
+        `Enum member names cannot start with lowercase 'a' through 'z'. Instead of using ` +
```

Comment:
It's specifically `'a' through 'z'`, because JavaScript can be written in any language (even those which do not have the concept of uppercase/lowercase letters), and our restriction is purely for 'a' through 'z'.

Reply:
Troll: It is a valid flow enum.
```js
enum A of string { е = "" }
```